js/components/calendar.vue

```diff
@@ -220,7 +216,7 @@ export default {
         },
 
         isMonthDisabled(idx) {
-            const month = moment().month(idx).year(this.currentYear);
+            const month = moment().month(idx).year(this.current.year());
```

Comment:
this should be `const month = this.current.clone().month(idx)`

Reply:
Agreed but non-significant here I think.